src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -770,6 +770,12 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
       else View.Empty
     }
 
+  /** Alias for `prependAll` */
+  def concat[B >: A](suffix: IterableOnce[B]): CC[B] = appendAll(suffix)
```

Comment:
Do you have an example of situation where one would want to have different implementations for `appendAll` and `concat`?

btw I just noticed that the scaladoc is wrong for this method (should be `appendAll` instead of `prependAll`).